lib/audio_service.dart

```diff
@@ -1434,9 +1444,14 @@ class AudioServiceBackground {
   /// SendPort/ReceivePort API. Please consult the relevant documentation for
   /// further information.
   static void sendCustomEvent(dynamic event) {
-    SendPort sendPort =
-        IsolateNameServer.lookupPortByName(_CUSTOM_EVENT_PORT_NAME);
-    sendPort?.send(event);
+    if (kIsWeb) {
+      print('Sending event: $event');
+      AudioService._customEventSubject.add(event);
+    } else {
+      SendPort sendPort =
+          IsolateNameServer.lookupPortByName(_CUSTOM_EVENT_PORT_NAME);
+      sendPort?.send(event);
+    }
```

Comment:
@hacker1024 This seems to be a pretty simple solution for getting the custom event stream to work. I imagine the same thing would work on MacOS.

Reply:
Yup, I didn't even think of doing it that way. I'll rebase on this branch and build on your work, so we won't have merge conflicts. Thanks!